packages/rest-specs/.gitignore

```diff
@@ -0,0 +1 @@
+cadl-ranch-coverage.json
```

Comment:
should delete this gitignore for now too

Reply:
Removed this file for now